app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
that could be.  I noticed the comments assumed they were about the top, removed the whitespaces at the top and then saw the comments stayed on the PR :)